src/Stratis.Bitcoin/Builder/FullNodeFeatureExecutor.cs

```diff
@@ -51,42 +51,25 @@ public FullNodeFeatureExecutor(IFullNode fullNode, ILoggerFactory loggerFactory)
         /// <inheritdoc />
         public void Initialize()
         {
-            try
-            {
-                this.Execute(service => service.ValidateDependencies(this.node.Services));
-                this.Execute(service => service.Initialize());
-            }
-            catch (Exception ex)
-            {
-                this.logger.LogError("An error occurred starting the application: {0}", ex);
-                throw;
-            }
+            this.Execute(service => service.ValidateDependencies(this.node.Services));
+            this.Execute(service => service.Initialize());
         }
 
         /// <inheritdoc />
         public void Dispose()
         {
-            try
-            {
-                this.Execute(feature => feature.Dispose(), true);
-            }
-            catch (Exception ex)
-            {
-                this.logger.LogError("An error occurred stopping the application", ex);
-                throw;
-            }
+            this.Execute(feature => feature.Dispose(), true);
         }
 
         /// <summary>
         /// Executes start or stop method of all the features registered with the associated full node.
         /// </summary>
         /// <param name="callback">Delegate to run start or stop method of the feature.</param>
         /// <param name="reverseOrder">Reverse the order of which the features are executed.</param>
-        /// <remarks>This method catches exception of start/stop methods and then, after all start/stop methods were called
-        /// for all features, it throws AggregateException if there were any exceptions.</remarks>
+        /// <remarks>This method catches exception of start/stop methods and then, after all start/stop methods were called for all features.</remarks>
```

Comment:
Incomplete comment

Reply:
In fact, you can remove this comment now.

Reply:
fixed